### ISSUE TITLE
Do you have a Generic content example?

### ISSUE THREAD

**Issue by SoftWebMike** [PiranhaCMS/piranha.core#1504]
Hey Håkan - have just started using Piranha and loving it!

I'm trying out rc2 and am eager to see the generic content content stuff. I can see in the source that there's a manager page for groups etc, was wondering if you had a quick example somewhere on how to preview it...? 

I managed to get as far as this, but I was not able to create a new "Product" from the management screen - keeps flagging a console error "Cannot read property 'create' of undefined... I'm guessing I have probably missed something

`    [ContentGroup(Title = "Content", Id = "Content")]
    [ContentType(Group = "Content", Id ="Product",Title = "Title")]
    public class ProductContent : Content<ProductContent>
    {
        [Region(Title = "Zoom")]
        public TextField SomeText{ get; set; }
    }`

Cheers and great job!

**Comment by tidyui** [comment]
Hi there! We're working on the new documentation sections for `9.0`, so it's not strange that you haven't found information about it. There's some examples on it in our repo in the `RazorWeb` example:

https://github.com/PiranhaCMS/piranha.core/tree/master/examples/RazorWeb/Models/Content

In short the key concept is that a `Group` equals an abstract base class, for example:

~~~
[ContentGroup(Title = "Products", Icon = "fas fa-hammer")]
public abstract class Product<T> : Content<T>, ICategorizedContent, ITaggedContent
    where T : Product<T>
{
}
~~~

The content types available for this group is then the classes that inherit from this base class, for example:

~~~
[ContentType(Title = "Standard Product")]
public class StandardProduct : Product<StandardProduct>
{
    [Region(Title = "All fields")]
    public Regions.AllFields AllFields { get; set; }
}
~~~

Best regards

**Comment by SoftWebMike** [comment]
Hey Håkan,

Thought that was the case ;)  Thanks for the reply!

Mike

**Comment by tidyui** [comment]
I updated my example as taxonomies for content will not be included in `9.0`. We're planning on releasing this in a minor release.

**Comment by hylinux** [comment]
@tidyui 

may I know what's purpose about the ContentGroup and Content ?
I read the source code: https://github.com/PiranhaCMS/piranha.core/tree/master/examples/RazorWeb,  look like the contentgroup and content were not used in the example code.

thanks
